### DISCUSSION TITLE
[Kavita] Ability to use KOMF to change volume/chapter covers instead of just series title

### DISCUSSION THREAD

**Discussion by Sonny-Crockett** [Kareadita/Kavita#2939] [Ideas]
### Idea Description

Either I am not doing this correctly, but I seem to only be able to add series level metadata using komf and tried three different providers. It will change the series thumbnail but even with komf listening I cannot trigger any changes or even downloads for the chapters/volumes. 

This appears to be related:
[https://github.com/Snd-R/komf/issues/140](url)

### Idea Category

User Experience

### Duration of Using Kavita

1 week

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
This was implemented from my standpoint and will be available from v0.8.3. 

https://github.com/Kareadita/Kavita/pull/3112

**Comment by majora2007** [comment]
Komf has also delivered this

**Comment by Sonny-Crockett** [comment]
Awesome work! Thanks to you both.  👍
